docs/sources/alerting/alerting-rules/create-grafana-managed-rule.md

```diff
@@ -236,6 +236,10 @@ To do this, you need to make sure that your alert rule is in the right evaluatio
    Pausing stops alert rule evaluation and doesn't create any alert instances.
    This is different to [mute timings](ref:mute-timings), which stop notifications from being delivered, but still allows for alert rule evaluation and the creation of alert instances.
 
+1. Set the time threshold for alerts firing.
+
+You can set the minimum amount of time that an alert remains firing after the breached threshold expression no longer returns any results. Select "none" stop an alert from firing immediately after the breach threshold is cleared. This sets an alert to a "Recovering" state for the duration of time set here. The Recovering state can be used to reduce noise from flapping alerts.
```

Comment:
```suggestion
You can set the minimum amount of time that an alert remains firing after the breached threshold expression no longer returns any results. This sets an alert to a "Recovering" state for the duration of time set here. The Recovering state can be used to reduce noise from flapping alerts. Select "none" stop an alert from firing immediately after the breach threshold is cleared.
```

Reply:
I reordered the text because it was a bit confusing. First, we say "Select none to stop the alert from firing immediately" (which means the state won't change to Recovering but to Normal), and then we immediately say "This sets an alert to a Recovering state," which is not accurate in this context.